src/Controller/CookieController.php

```diff
@@ -43,7 +45,13 @@ public function __construct(KillSwitch $kill_switch)
    * @return \Symfony\Component\HttpFoundation\RedirectResponse
    */
   public function setCookie() {
+    $cookie_domains = array_map('trim', explode("\n", $this->config->get('cookie_domains')));
     setcookie("analytics_disable", TRUE, 0, '/');
+    if (!empty($cookie_domains)) {
```

Comment:
jesli $cookie_domains bedzie [] to nie ma potrzeby sprawdzac !empty, foreach ignoruje puste arrayki i sie nie wykona.